### PR TITLE
fix fossa

### PR DESCRIPTION
**Pull request by gy95**
Signed-off-by: guoyao <1015105054@qq.com>

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by RainbowMango** [comment]
I'm afraid it's not the reason.

**Comment by gy95** [comment]
seems passed, but i don't know why it works.

**Comment by RainbowMango** [comment]
Can you also try with the `https://github.com/fossas/fossa-action`?

File: .github/workflows/fossa.yml [hunk 1]
```diff
@@ -12,6 +14,12 @@ jobs:
       - name: checkout code
         uses: actions/checkout@v2
       - name: Run FOSSA scan and upload build data
-        uses: fossa-contrib/fossa-action@v1
-        with:
-          fossa-api-key: 7ac0174603e25cb0a4b3ebd2e131a89f
+        #uses: fossa-contrib/fossa-action@v1
```
Comment (RainbowMango) [inline .github/workflows/fossa.yml:17]:
```suggestion
        uses: fossas/fossa-action@v1
```
Try this one. Seems this is the official fossa-action.

Comment (gy95) [reply .github/workflows/fossa.yml]: works well

File: .github/workflows/fossa.yml [hunk 2]
```diff
@@ -3,7 +3,9 @@ on:
   # Run this workflow every time a new commit pushed to upstream/fork repository.
   # Run workflow on fork repository will help contributors find and resolve issues before sending a PR.
   push:
+    branches: [master]
   pull_request:
+    branches: [master]
```
Comment (RainbowMango) [inline .github/workflows/fossa.yml:8]: Why it should be restricted to the `master` branch?

File: .github/workflows/fossa.yml (hunk 1 above)
Comment (RainbowMango) [reply .github/workflows/fossa.yml]: Nice.

File: .github/workflows/fossa.yml (hunk 2 above)
Comment (gy95) [reply .github/workflows/fossa.yml:8]: only trigger ci checks when PR is submitted to karmada-io/karmada master branch.

Comment (RainbowMango) [reply .github/workflows/fossa.yml:8]: You might didn't see the comments above:
```
# Run this workflow every time a new commit pushed to upstream/fork repository.
  # Run workflow on fork repository will help contributors find and resolve issues before sending a PR.
```

Comment (gy95) [reply .github/workflows/fossa.yml]: done.

**Pull request merged**